src/services/policies/index.ts

```diff
@@ -78,6 +78,24 @@ const mockGetPolicies = async (
             label: "$"
           }
         }
+      },
+      {
+        category: "vouchers",
+        name: "Vouchers",
+        order: 4,
+        quantity: { period: 1, limit: 1 },
+        identifiers: [
+          {
+            label: "Voucher code",
+            textInput: { visible: true, disabled: true, type: "STRING" },
+            scanButton: {
+              visible: true,
+              disabled: false,
+              type: "QR",
```

Comment:
actually, should we keep it as `type: "BARCODE"` haha

Reply:
Oh right, can also~~ Not much impact now cuz it's not supported anyway HAHA

Reply:
HAHA, its okk~~ at least we make it accurate 😂  cause now we'll send the type into the db also haha

Reply:
Ohhh, this one won't get sent into the db. It only gets activated when we run `npm run start:mock`, which mocks the backend~